scripts/tests/scripts_regression_tests.py

```diff
@@ -1299,6 +1299,12 @@ def test_update_acme_tests_test_mods(self):
         #                  msg="COMMAND SHOULD HAVE WORKED\ncs.status output:\n%s\n\nerrput:\n%s\n\ncode: %d" % (output, errput, stat))
 
 ###############################################################################
+class W_TestCreateTest(TestCreateTestCommon):
+###############################################################################
+    def test_create_test_longname(self):
```

Comment:
You're right. The class I was thinking of was TestTestScheduler. That said, this is probably the kind of test I'd put in K_TestCimeCase.